### ISSUE TITLE
Viewport height change doesn't change 'showing' rows

### ISSUE THREAD

**Issue by mmuruev** [DataTables/Scroller#72]
If change height '.dataTables_scrollBody' then no additional rows appears plugin still thinks like viewport stay the same.

**Comment by DataTables** [comment]
Yes - the high is not dynamic. There should be no mention of that in the documentation or examples. If there is, could you let me know so I can remove it please.

**Comment by mmuruev** [comment]
So this is not a bug? I mean with Layout and  PageResize plugins this extensions will never work?

**Comment by DataTables** [comment]
Sorry I hadn't realised that you were using PageResize / ScrollResize.

It isn't something I've tested to be honest, so no, I don't think it would work at the moment. It would require the scrolling position to be updated.

**Comment by mmuruev** [comment]
I did it and it seems works, but have internal 'buffer' issue. If page is huge enough then 'tail' of the data which less then on page size never loaded. Basically this
` if ( iTopRow != this.s.dt._iDisplayStart )`
statement will be always 'false' both values = 0;  Besides not really understands the idea of counting by pixels not by rows. I think just track size 'buffer' and reload it near the bound will work in any case, so case like this never can appear.

**Comment by scsloan** [comment]
Running into this as well. Any plan to fix this?

**Comment by DataTables** [comment]
Its not something I'm working on at the moment, so if someone else would like to look into it, they are welcome to.
